experimental/packages/exporter-trace-otlp-proto/test/node/OTLPTraceExporter.test.ts

```diff
@@ -22,13 +22,13 @@ import * as http from 'http';
 import * as sinon from 'sinon';
 import { Stream, PassThrough } from 'stream';
 import * as zlib from 'zlib';
-import { OTLPTraceExporter } from '../src';
+import { OTLPTraceExporter } from '../../src/platform/node';
```

Comment:
Small suggestion: It might be worth keeping this as "../../src" so that it would be indirectly referencing the exporter via the "index.ts" (which is node by default) -- unless you need to access any "internal" overrides.

Reply:
@MSNev made this change.